indico/modules/events/templates/reviews/_common.html

```diff
@@ -70,3 +70,28 @@
 {% macro render_review_state(review) %}
     <span class="bold underline semantic-text {{ review.proposed_action.css_class }}">
 {% endmacro %}
+
+{% macro render_timeline_section(cf) %}
```

Comment:
> And what should I call the argument representing CFP or CFA that can be passed there ?  

I think we are calling them _Call for proposals_ (so `cfp`) as stated here: https://github.com/indico/indico/blob/master/indico/modules/events/models/reviews.py#L39
Not 100% sure though.